packages/peregrine/lib/talons/CategoryTree/__tests__/useCategoryTree.spec.js

```diff
@@ -266,4 +266,38 @@ describe('child categories', () => {
         const { childCategories } = log.mock.calls[0][0];
         expect(childCategories.has(1)).toEqual(false);
     });
+
+    test('Does not include child category when include_in_menu is falsy', () => {
+        const categoryId = 5;
+        useLazyQuery.mockImplementation(() => {
+            return jest.fn().mockReturnValue([
+                getNavigationMenu,
+                {
+                    data: {
+                        category: {
+                            ...result.data.category,
+                            children: [
+                                ...result.data.category.children,
+                                {
+                                    id: categoryId,
+                                    include_in_menu: 0,
```

Comment:
Because the default data `result` all has `include_in_menu: 1`, we have to augment/add a case that will check the false branch/path.